src/components/Results/Results.tsx

```diff
@@ -47,6 +48,16 @@ const Results: React.FC<Props> = ({ measureFile, htmls }) => {
                 </Grid>
               );
             })}
+          {results &&
+            detectedIssues.map((issue: R4.IDetectedIssue) => {
+              return (
+                <div>
```

Comment:
![image](https://user-images.githubusercontent.com/16297930/125801504-df61cd7e-9a8d-4498-b5f5-27e836e26f63.png)

You an do some digging into why React wants this, but the actual change to make is to add a `key` prop to the first child that appears inside a `.map`. This `key` should uniquely identify the element. In this case, you'd want to use the resource ID.

I see you're doing a fhirpath evaluate to grab the ID. Let's pull that out into a variable before the `return` statement, and use that as the `key` for the `div` as well.